src/createXcmTypes/util/sortMultiAssetsAscending.ts

```diff
@@ -13,32 +12,190 @@ import {
  */
 export const sortMultiAssetsAscending = (multiAssets: MultiAsset[]) => {
```

Comment:
It is likely that future versions of XCM will add new junction types, different arrays etc and this function is not going to be easy to maintain. We should have this into consideration if we want to keep this implementation of the sorting function.